aiohttp/web_protocol.py

```diff
@@ -639,6 +646,13 @@ def handle_error(
         information. It always closes current connection."""
         self.log_exception("Error handling request", exc_info=exc)
 
+        # some data already got sent, connection is broken
+        if request.writer.output_size > 0:
+            raise ConnectionError(
+                "Response is sent already, cannot send another response "
+                "with the error message"
+            )
```

Comment:
How about
```suggestion
            ) from exc
```

Reply:
No, `exc` is not related to the raised ConnectionError exception.
This ConnectionError is not caught by a user but intercepted by web_protocol internals and silently ignored.
I can send bare ConnectionError technically but prefer attaching a string annotation which can help in debugging in the future.

Plus, `exc` is logged already.